modules/fetching-vulnerability-definitions.adoc

```diff
@@ -5,8 +5,9 @@
 [id="fetching-vulnerability-definitions_{context}"]
 = Fetching vulnerability definitions
 
+//verify this info is still accurate
 In online mode, Central fetches the vulnerability definitions every 5 minutes from a single feed.
-This feed combines vulnerability definitions from upstream sources that include multiple Linux distributions and the National Vulnerability Database, and it refreshes every hour.
+This feed combines vulnerability definitions from upstream sources that include multiple Linux distributions, the National Vulnerability Database, and other sources, and it refreshes every hour.
 
 * The address of the feed is `\https://definitions.stackrox.io`.
 * You can change the default query frequency for Central by setting the `ROX_SCANNER_VULN_UPDATE_INTERVAL` environment variable:
```

Comment:
note: there is a message below this about a configurable update interval for Scanner. Let's make it clear this is not configurable in Scanner V4